<launch>
  <!-- Read arguments -->
  <arg name="old_frame" default="world"/>
  <arg name="new_frame" default="base_link"/>
  <arg name="input_topic" default="/mavros/imu/data"/>






  <node name="tfpub"
        pkg="rov_control_uwsim"
        type="tfpub.py"
        output="screen"
        respawn="false">
    <!-- Set ROS parameters -->
    <param name="old_frame" value="$(arg old_frame)"/>
    <param name="new_frame" value="$(arg new_frame)"/>
    <param name="input_topic" value="$(arg input_topic)"/>
  </node>
</launch>
